dataprofiler/profilers/base_column_profilers.py

```diff
@@ -267,6 +267,7 @@ def __init__(self, name: Optional[str]) -> None:
         # Number of values that match the column type. eg. how many floats match
         # in the float column
         self.match_count: int = 0
+        self.sample_size: int  # inherited from BaseColumnProfiler
```

Comment:
`self.sample_size` wasn't being detected by mypy to be inherited from `BaseColumnProfiler`. This was causing problems downstream where mypy didn't have any type information on `self.sample_size`. I suspect the `with_metaclass` line above is causing this issue.

Reply:
awesome